### PR TITLE
respawn works?

### PR DESCRIPTION
**Pull request by catornot**

### CODE REVIEW

**Review by xamionex — CHANGES_REQUESTED** [review]
this works

File: Northstar.Coop/mod/scripts/vscripts/sp/_base_gametype_sp.gnut [hunk 1]
```diff
@@ -454,7 +456,6 @@ function PostDeathThread_SP( entity player, damageInfo )
 				if (p != null){
 				SendHudMessage( p , Coop_Dead_Player + " Fucked up, laugh at them." , -1, 0.4, 255, 255, 0, 0, 0.15, 6, 0.15 )
 				//ClientCommand( p , "RespawnNowSP" )
-				RespawnPlayer( p )
```
Comment (xamionex) [inline Northstar.Coop/mod/scripts/vscripts/sp/_base_gametype_sp.gnut:462]: change this from p to player, then it works

**Review by xamionex — CHANGES_REQUESTED** [review]

File: Northstar.Coop/mod/scripts/vscripts/sp/_base_gametype_sp.gnut (hunk 1 above)
Comment (catornot) [reply Northstar.Coop/mod/scripts/vscripts/sp/_base_gametype_sp.gnut:462]: yeah did the same thing

Comment (xamionex) [reply Northstar.Coop/mod/scripts/vscripts/sp/_base_gametype_sp.gnut:462]: > yeah did the same thing

yeye but commit so i can do PR

**Comment by catornot** [comment]
I can check if it works, but looking at the code. It should.

File: Northstar.Coop/mod/scripts/vscripts/sp/_base_gametype_sp.gnut (hunk 1 above)
Comment (catornot) [reply Northstar.Coop/mod/scripts/vscripts/sp/_base_gametype_sp.gnut:462]: Why is the line being removed? 
We need it right?

**Comment by catornot** [comment]
I will make another pr tmr If I even get it working.
this one isn't going to work.

**Pull request closed without merging**